CHANGELOG.md

```diff
@@ -50,8 +50,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 
 ### Removed
+- AnyToMetadataClass and AnyToMetadata type. Neither was really used anywhere
```

Comment:
I don't think this is necessary as these weren't in the public interface.

Other than that lgtm, and `make run-ci-actions` is passing locally

Reply:
Agreed